cmd/buildah/commit.go

```diff
@@ -93,9 +93,9 @@ func commitCmd(c *cobra.Command, args []string, iopts commitInputOptions) error
 		return errors.Errorf("too many arguments specified")
 	}
 	image := args[0]
-	compress := imagebuildah.Uncompressed
-	if c.Flag("disable-compression").Changed && !iopts.disableCompression {
-		compress = imagebuildah.Gzip
+	compress := imagebuildah.Gzip
+	if iopts.disableCompression {
+		compress = imagebuildah.Uncompressed
```

Comment:
Maybe I'm light on tea, bu the man page says: 

```
-disable-compression, -D

Don't compress filesystem layers when building the image unless it is required by the location where the image is being written. This is the default setting, because image layers are compressed automatically when they are pushed to registries, and images being written to local storage would only need to be decompressed again to be stored. Compression can be forced in all cases by specifying --disable-compression=false.
```

So I think the default should be Uncompressed at line 96 and then `if !iopts.disableCompression {` (note changing to false), then setting compression to Gzip.

@nalind, WDYT?

Reply:
FWIW, I think the `c.Flag("disable-compression").Changed is probably not needed.

Reply:
I went with the same logic in push.go, to make all three consistent